beep/generate_protocol.py

```diff
@@ -167,27 +168,29 @@ def generate_protocol_files_from_csv(csv_filename, output_directory=None):
             filename = os.path.join(output_directory, "settings", filename)
 
         else:
-            warnings.warn("Unsupported file template {}, skipping.".format(template))
-            result = "error"
-            message = {
+            failure = {
                 "comment": "Unable to find template: " + template,
                 "error": "Not Found",
             }
+            file_generation_failures.append(failure)
+            warnings.warn("Unsupported file template {}, skipping.".format(template))
+            result = "error"
             continue
 
         logger.info(filename, extra=s)
         if not os.path.isfile(filename):
             protocol.to_file(filename)
-            new_files.append(filename)
+            successfully_generated_files.append(filename)
             names.append(filename_prefix + "_")
 
         elif ".sdu" in template:
+            failure = {
+                "comment": "Unable to find template: " + template,
```

Comment:
Is there a reason not to keep the previous `Schedule file generation is not yet implemented` message?

Reply:
no, that was a mistake good catch!